.github/workflows/cluster_cleanup.yml

```diff
@@ -99,6 +81,22 @@ jobs:
           for resource in $(az resource list -g ${{ env.RESOURCE_GROUP }} --query "[?starts_with(name, '${{ env.CLUSTER_PREFIX }}')].id" -o tsv); do
             az resource delete -v --id $resource --verbose
           done
+      - name: Delete schema registries
+        run: |
+          for resource in $(az resource list -g ${{ env.RESOURCE_GROUP }} --query "[?starts_with(name, '${{ env.REGISTRY_PREFIX }}')].id" -o tsv); do
+            az resource delete -v --id $resource --verbose
+          done
+      - name: Delete storage accounts
+        run: |
+          for resource in $(az resource list -g ${{ env.RESOURCE_GROUP }} --query "[?starts_with(name, '${{ env.STORAGE_PREFIX }}')].id" -o tsv); do
+            az resource delete -v --id $resource --verbose
+          done
+      - name: Delete your custom stuff
+        if: ${{ env.CUSTOM_PREFIX }}
```

Comment:
This is dangerous but I understand why it's necessary - perhaps in the future we should add some kind of checking for a "do not delete" tag of sorts?

Reply:
future thing yea but we like to live dangerously